Update readme (#82)
microsoft/fabric-samples: README.md

![Microsoft Logo](./media/microsoft-logo-small.png)

# Microsoft Fabric Samples

Welcome to the repository of Microsoft Fabric samples! This collection is designed to help you explore and learn about Microsoft Fabric, offering practical examples and resources. Whether you're new to Microsoft Fabric or looking to deepen your understanding, you'll find valuable insights and tools here.

## What's Inside

This repository is organized into different sections to help you navigate through various samples and resources easily.

### Documentation Samples

- **Location:** `docs-samples` folder.

> - **Description:** These samples support the Microsoft Fabric official documentation and Microsoft Learn modules. They include both code examples and sample data.

Reviewer: ```suggestion
- **Description:** These samples are referenced within the Microsoft Fabric official documentation and Learn training content. They include both code examples and sample data.
```

### Feature Samples

- **Location:** `feature-samples` folder.

> - **Description:** Focus on specific features of Microsoft Fabric, showcasing how to leverage them in your projects.

Reviewer: ```suggestion
- **Description:** These samples focus on specific features of Microsoft Fabric, demonstrating how to leverage them in your projects.
```

### Fabric Readiness Repository

The [Fabric Readiness Repository](https://github.com/microsoft/Fabric-Readiness) contains presentation decks for various purposes, including user groups, online presentations, in-person conferences, and customer discussions. These resources come with comprehensive notes for presenters.

### Fabric Notes

For a visually engaging learning experience, check out [Fabric Notes](https://aka.ms/fabricnotes). These illustrated notes break down Microsoft Fabric's core concepts into easy-to-understand visuals.
  
  ![Fabric Notes Sample](https://microsoft.github.io/fabricnotes/images/notes/01-welcome.png)

## Contributing

While we're currently not accepting contributions from external contributors, we're eager to hear about the samples you'd like to share. If you have a sample proposal:

1. Ensure any datasets used are licensed appropriately for sharing.
2. Open a draft pull request with the following
    - A new folder under `community-samples` with a short name for your sample
    - A `readme.md` sample describing the objective of the sample

>     - If the sample code is not yet written, please wait for confirmation in the PR disucssion before engaging time to create the sample.

Reviewer: Will there be some evaluation criteria/best practices before merging the PRs?
